res/style.css

```diff
@@ -169,26 +169,23 @@ body, #root, .profileViewer {
 
 .treeViewAppendageColumn {
   margin-left: 10px;
-  color: #999;
+  opacity: 0.6;
 }
 
-.treeViewRow.even, .treeViewRow.even > .treeViewFixedColumn {
```

Comment:
We don't need to specify the fixed column, as far as I know they don't override this value